src/irmin-pack/pack_value.ml

```diff
@@ -2,35 +2,57 @@ open! Import
 include Pack_value_intf
 
 module Kind = struct
-  type t = Commit | Contents | Inode_v0_unstable | Inode_v0_stable
+  type t =
+    | Commit
+    | Contents
+    | Inode_v0_unstable
+    | Inode_v0_stable
+    | Inode_v1_root
+    | Inode_v1_nonroot
 
   let to_magic = function
     | Commit -> 'C'
     | Contents -> 'B'
     | Inode_v0_unstable -> 'I'
     | Inode_v0_stable -> 'N'
+    | Inode_v1_root -> 'R'
+    | Inode_v1_nonroot -> 'O'
```

Comment:
I picked these new magics semi-randomly. It's now or never to change these!

Reply:
Personally I'm happy with semi-random choices. As of https://github.com/mirage/irmin/commit/a80d01883605d71c3ecdd20fcec9263bbeba8991 we should see the longer names in our logs anyway.